i18n/de.yaml

```diff
@@ -283,16 +283,16 @@
   translation: "Sia Blockchain Explorer"
 
 - id: apps_minebox
-  translation: "Next generation NAS powered by Sia" # TODO: translate
+  translation: "NAS der nächsten Generation, ermöglicht durch Sia"
 
 - id: apps_siacluster
-  translation: "Data farm monitoring and management tool for Sia" # TODO: translate
+  translation: "Überwachung und Verwaltung eines Sia Clusters"
 
 - id: apps_gnomeshell
-  translation: "Sia Cloud Storage Extension for GNOME Shell" # TODO: translate
+  translation: "Sia Cloud Speicher Erweiterung für die GNOME Shell"
 
 - id: apps_siapulse
-  translation: "Analytics tool for Sia" # TODO: translate
+  translation: "Analysewerkzeug für Sia"
```

Comment:
I think "Analysetool" would sound nicer. I think "Tool" by itself is even in the Duden by now.